src/Corax/IndexWriter.cs

```diff
@@ -719,24 +719,34 @@ void ExactInsert(ReadOnlySpan<byte> value)
         private static unsafe ByteStringContext<ByteStringMemoryCache>.InternalScope CreateNormalizedTerm(ByteStringContext context, ReadOnlySpan<byte> value,
             out Slice slice)
         {
-            ulong hash = 0;
+            ByteStringContext<ByteStringMemoryCache>.InternalScope scope;
```

Comment:
Any idea why the Dynamic Fields changes could have an impact here and cause this to malfunction? cc @maciejaszyk @ayende

Reply:
This is solved on the newer null aware encoding algorithm